srcalc/src/test/java/gov/va/med/srcalc/web/view/admin/EditRiskModelTest.java

```diff
@@ -49,12 +55,31 @@ public final void testEditRiskModel()
         
     	editRiskModel.setModelName( "NewModelName" );
     	
-        assertEquals( editRiskModel.getModelName(), "NewModelName" );
-        assertEquals( editRiskModel.getMaxDisplayNameLength(), DisplayNameConditions.DISPLAY_NAME_MAX );
+        assertEquals( "NewModelName", editRiskModel.getModelName() );
+        assertEquals(  DisplayNameConditions.DISPLAY_NAME_MAX, editRiskModel.getMaxDisplayNameLength() );
         
         editRiskModel.applyChanges();
         
         RiskModel rm = editRiskModel.getRiskModel();
-        assertEquals("NewModelName", rm.getDisplayName() );
+        assertEquals( "NewModelName", rm.getDisplayName() );
+    }
+
+    @Test
+    public final void testGetTermSummaries()
+    {
+    	final EditRiskModel editRiskModel = createEditRiskModel( "Thoracic 30-day Mortality" );
```

Comment:
It's fine if the method is left with a String parameter, but I think the literal string should be replaced with a constant that stores the string. That way if you want to change something in the string, you just reuse the constant. Keeps duplication down.